src/scene/scene_forwardrenderer.js

```diff
@@ -631,7 +636,31 @@ pc.extend(pc, function () {
             drawCalls.sort(sortDrawCalls);
 
             // Render a depth target if the camera has one assigned
-            if (camera._depthTarget) {
+            if (camera._renderDepthRequests) {

```

Comment:
Again it's not worse than the original code
https://github.com/playcanvas/engine/blob/master/src/scene/scene_forwardrenderer.js#L634